src/main/java/racinggame/domain/business/RacingGame.java

```diff
@@ -0,0 +1,41 @@
+package racinggame.domain.business;
+
+import static racinggame.domain.business.CarFactory.createCars;
+import static racinggame.domain.util.RandomUtil.generateInt;
+
+import java.util.List;
+import racinggame.domain.model.Car;
+import racinggame.domain.model.RacingResult;
+
+public class RacingGame {
+    
+    public static final int DEFAULT_COUNT = 1;
+    private int raceCount;
+    private final List<Car> cars;
+    
+    public RacingGame(String carsName) {
+        this(carsName, DEFAULT_COUNT);
+    }
+    
+    public RacingGame(String carsName, int raceCount) {
+        this.cars = createCars(carsName);
+        this.raceCount = raceCount;
+    }
+    
+    public RacingResult race() {
+        carsRace();
+        this.raceCount --;
+        return new RacingResult(this.cars);
+    }
```

Comment:
```suggestion
    public void race() {
        carsRace();
        this.raceCount --;
    }
    
    public RacingResult raceResult() {
        return new RacingResult(this.cars);
    }
```
재영님이 구현한 코드와 아래와 같이 분리해 구현했을 때 각각의 장단점은 무엇인가?
어떻게 구현하는 것이 좋을까 고민해 보는 계기가 되면 좋겠음.